models/labels/addresses/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql

```diff
@@ -27,9 +28,9 @@ WITH sender_transfer_rates AS (
                     WHEN EXISTS (SELECT 1 FROM {{ ref('nft_transfers') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1
                 ELSE 0 END) AS num_token_tfer_txs
     
-            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('dex_trades') }} r WHERE t.hash = r.tx_hash AND t.block_time = r.block_time AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_dex_trade_txs
-            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('perpetual_trades') }} r WHERE t.hash = r.tx_hash AND t.block_time = r.block_time AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_perp_trade_txs
-            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('nft_trades') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_nft_trade_txs
+            , 0 /*SUM(CASE WHEN EXISTS (SELECT 1 FROM [[ ref('dex_trades') ]] r WHERE t.hash = r.tx_hash AND t.block_time = r.block_time AND blockchain = 'optimism') THEN 1 ELSE 0 END)*/ AS num_dex_trade_txs
+            , 0 /*SUM(CASE WHEN EXISTS (SELECT 1 FROM [[ ref('perpetual_trades') ]] r WHERE t.hash = r.tx_hash AND t.block_time = r.block_time AND blockchain = 'optimism') THEN 1 ELSE 0 END)*/ AS num_perp_trade_txs
+            , 0 /*SUM(CASE WHEN EXISTS (SELECT 1 FROM [[ ref('nft_trades') ]] r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END)*/ AS num_nft_trade_txs
```

Comment:
we've seen odd behaviors in orchestration when using sql comments on jinja refs, related to state comparison between manifest files. it would help to wrap these in jinja comments instead.
`{# a comment #}`

Reply:
do you still want to proceed and merge these with the comments, or just mark this as blocked and wait?

Reply:
Is there a better way to comment out un-migrated models? I can remove the ref as well, so that there's no confusion